docs/api-fields.md

```diff
@@ -88,23 +88,23 @@ A checkbox.
 
```

Comment:
As reading this file in this form hurts your eyes, take a look at least on 'feedbackable', 'inline' and 'labelBefore' props, as  have not found sufficient sources on this subject.